index.html.in

```diff
@@ -31,6 +31,7 @@
 
         <!--[if lt IE 9]> <script type="text/javascript" src="./js/compat.js"></script> <![endif]-->
 @include_head@
+	<!-- qunit head -->
```

Comment:
`qunit-head`? I'm hoping this only happens with `make test`, we don't need everyone running tests ;)

Reply:
@sushain97 The `<!-- qunit head -->` and `<!-- qunit-tests -->`only get injected when running on `qunit-test.html`.